core/src/usb.py

```diff
@@ -54,7 +63,8 @@
 bus.add(iface_wire)
 if __debug__:
     bus.add(iface_debug)
-    bus.add(iface_vcp)
 if not utils.BITCOIN_ONLY:
-    if not __debug__ or utils.EMULATOR:
-        bus.add(iface_webauthn)
+    bus.add(iface_webauthn)
```

Comment:
presumably this is the place I would need to change to get the VCP interface on a debug build?

what happens when the webauthn app boots and the webauthn interface is not registered? will the device start up fine?

Reply:
I guess it would start fine, but webauthn would not work.